src/components/instances/new/MetaLocal.js

```diff
@@ -50,7 +51,23 @@ function MetaLocal() {
 				setFormState({ error: 'usernames must have only letters, underscores, and hyphens' });
 			} else if (instance_name.length && user.length && pass.length && host.length && port.length) {
 				try {
-					const currentUser = await userInfo({ auth: { user, pass }, url, is_local: true, customer_id });
+					let login_not_supported;
+					let currentUser;
+					if (pass) {
+						const login_result = await login({ auth: { user, pass }, url });
+						if (login_result.error) {
+							setFormState({ error: 'Login failed. Using instance credentials?' });
+						} else {
+							currentUser = await userInfo({ url });
+							if (currentUser.error) {
+								login_not_supported = true;
+							}
+							if (login_not_supported)
+								currentUser = await userInfo({ auth: { user, pass }, url, is_local: true, customer_id });
```

Comment:
Not sure what's going on here, but this seems like unnecessary complexity.

```suggestion
							if (currentUser.error) {
								currentUser = await userInfo({ auth: { user, pass }, url, is_local: true, customer_id });
							}
```